packages/datadog-instrumentations/src/graphql.js

```diff
@@ -350,6 +350,11 @@ function finishResolvers ({ fields }) {
   })
 }
 
+addHook({ name: '@graphql-tools/executor', file: 'cjs/execution/execute.js', versions: ['>=0.0.14'] }, execute => {
```

Comment:
Does this mean that ESM is not supported?

Reply:
Haven't tested it with ESM yet. But this is more of a feature request to support graphql-yoga v3 https://github.com/DataDog/dd-trace-js/issues/2884 which uses @graphlql-tools/executor as it's executor rather the the graphql executor